qvisor/src/elf_loader.rs

```diff
@@ -166,6 +168,113 @@ impl KernelELF {
         return Ok(entry);
     }
 
+    #[cfg(feature = "cc")]
```

Comment:
how different is it to load the kernel with cc enabled? The repetition is pretty heavy here.

Reply:
Not much, I can merge them.